src/panels/config/repairs/ha-config-repairs.ts

```diff
@@ -43,41 +40,42 @@ class HaConfigRepairs extends LitElement {
           count: this.total || this.repairsIssues.length,
         })}
       </div>
-      <mwc-list>
-        ${issues.map(
-          (issue) => html`
-            <ha-list-item
-              twoline
-              graphic="medium"
+      <ha-list-new>
```

Comment:
You will need to rename ha-list-new to ha-md-list and ha-list-item-new to ha-md-list-item since #22015 went in right now.